docs/core/compatibility/core-libraries/5.0/environment-osversion-returns-correct-version.md

```diff
@@ -9,14 +9,16 @@ ms.date: 11/01/2020
 
 ## Change description
 
-In previous .NET versions, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns an OS version that may be incorrect when an application runs under Windows compatibility mode. For more information, see [GetVersionExA function remarks](/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversionexa#remarks).
+In previous .NET versions, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns an OS version that may be incorrect when an application runs under Windows compatibility mode. For more information, see [GetVersionExA function remarks](/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversionexa#remarks). On macOS, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns the underlying Darwin kernel version.
 
-Starting in .NET 5.0, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns the actual version of the operating system.
+Starting in .NET 5.0, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns the actual version of the operating system for Windows and macOS.
```

Comment:
maybe add an example to make it super clear what changed:
```suggestion
Starting in .NET 5.0, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns the actual version of the operating system for Windows and macOS, e.g. instead of `19.6.0.0` it returns `10.15.7` on macOS.
```

Reply:
If you include that, please spell out "for example,". I would also make it a separate sentence.

Reply:
I decided to put a table of the differences to show both OSes. PTAL